main.go

```diff
@@ -100,23 +102,25 @@ func main() {
 	setupLog := log.Log
 
 	var (
-		configFile string
-		err        error
+		metricsAddr          string
+		enableLeaderElection bool
+		probeAddr            string
+		err                  error
 	)
-	flag.StringVar(&configFile, "config", "",
-		"The operator will load its initial configuration from this file. "+
-			"Omit this flag to use the default configuration values. "+
-			"Command-line flags override configuration from this file.")
+	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
+	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
+		"Enable leader election for controller manager. "+
+			"Enabling this will ensure there is only one active controller manager.")
 	flag.Parse()
 
-	options := ctrl.Options{Scheme: scheme}
-
-	if configFile != "" {
-		options, err = options.AndFrom(ctrl.ConfigFile().AtPath(configFile))
```

Comment:
`ComponentConfig` has been deprecated (https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.15.0). We can either use a custom implementation that sets Manager.Options directly or move back to configuration via arguments. 

Moving back to configuration using arguments as it's the easier option and we do not have many configurations in general